packages/@aws-cdk/aws-lambda/test/test.lambda.ts

```diff
@@ -911,6 +911,220 @@ export = {
       test.done();
     },
 
+    'default function with Active tracing'(test: Test) {
+      const stack = new cdk.Stack();
+
+      new lambda.Function(stack, 'MyLambda', {
+          code: new lambda.InlineCode('foo'),
+          handler: 'index.handler',
+          runtime: lambda.Runtime.NodeJS610,
+          tracingMode: lambda.TracingMode.Active
+      });
+
+      expect(stack).toMatch(
```

Comment:
Use “expect(stack).to(haveResource(“ so test will be less heavy handed

Reply:
done